Dependencies/AndroidExtensions/Source/JavaWrappers.cpp

```diff
@@ -68,6 +108,25 @@ namespace java::lang
         return m_env->IsAssignableFrom(m_class, otherClass.m_class);
     };
 
+    jclass Class::JClass() const
+    {
+        return m_class;
+    }
+
+    void Class::JClass(jclass classObj)
+    {
+        if (m_class) {
+            m_env->DeleteGlobalRef(m_class);
+        }
+
+        m_class = nullptr;
+
+        if (classObj)
+        {
+            m_class = static_cast<jclass>(m_env->NewGlobalRef(classObj));
+        }
```

Comment:
It seems like this should be:
```suggestion
        m_class = classObj;

        if (m_class)
        {
            m_class = static_cast<jclass>(m_env->NewGlobalRef(classObj));
        }
```
Was there some reason we couldn't do it that way? Can't remember.

Reply:
No real reason. I've updated to your proposed format for both the Class and Object implementation.